ast.go

```diff
@@ -100,7 +100,7 @@ func (*Revoke) iGrantOrRevokeStatement() {}
 type Select struct {
 	Distinct         string
 	SelectColumnList SelectColumnList
-	From             TableExprList
+	From             TableExpr
```

Comment:
So a `From` is not a list anymore it is just a `TableExpr` (interface), and you can see all types of `TableExpr` that exist at [L188-L190](https://github.com/tablelandnetwork/go-sqlparser/pull/33/files#diff-0945956cd950d047e7fa866dcf34fee7d29cbce277d4e5e5cd621f2213f3be6cR189-R190). That is:
- a simple aliased table: `FROM table as t`
- a parenthesized table expression: `FROM (table)`
- a join: `FROM a JOIN B` or `FROM a, b`